src/overlays/bankid/spinner.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+interface IProps {
+    hasError: boolean;
+}
+
+export default ({ hasError }: IProps) =>
+    !hasError ? (
```

Comment:
I don't think a `hasError` property belongs on the spinner. An alternative is to use a more generic name, such as `visible` - but that might be unnecessary as well.

The pattern used earlier in this library, has the possibility to omit the `hasError` property all togehter during component use;
```javascript
...
<div>
    {hasError ? null : <Spinner />}
</div>
...
```

Reply:
I'll use that, much better.